packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -94,6 +95,7 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
     };
 
     const logoutHandler = async () => {
+        await setIsForcedToExitPnv(true);
```

Comment:
should we call `setIsForcedToExitPnv(true);` in all scenarios?
how about we check if the route is related to pnv flow, then only call `await setIsForcedToExitPnv(true);`

Reply:
yeap we could do that also 🙏